### ISSUE TITLE
P1 block diagram MICRO_SPI1_SS pin name needs a note added for clarification

### ISSUE THREAD

**Issue by dmiddlecamp** [particle-iot/docs#469]
this image: 
https://docs.particle.io/assets/images/p1-pinout1.png

![](https://docs.particle.io/assets/images/p1-pinout1.png)

Notice how the SPI_SS starts on PA4, when it should be for PC2.

**Comment by technobly** [comment]
The way that SPI is currently pinned out is correct, we do not use the PA4 SS pin in an hardware SPI driven sense.  SS is only user controlled as a GPIO, thus the default one is A2 (PC2) https://docs.particle.io/reference/firmware/photon/#spi

**Comment by dmiddlecamp** [comment]
So it's correct that DAC is labelled as MICRO_SPI_SS ?

**Comment by technobly** [comment]
Yes, we list out the P1 module pin names... but don't necessarily need to use MICRO_SPI1_SS in a hardware sense, but the DAC1 function of that pin was thought to be more important to route to A6/DAC pin at the time.  Rather than obfuscating the P1 module pin name, we chose to leave it as-is.  It's a bit confusing understandably, so perhaps we need to add a note to the P1 datasheet to help clarify this.

The location of SPI1_SS on A2 is done that way to be pin compatible with the Spark Core.

**Comment by technobly** [comment]
For P1 products, I suppose you could wire up PA4 as your SPI SS pin to be used in code as SPI.begin(DAC) or SPI.begin(A6) or SPI.begin(DAC1), but it would just use that pin as a GPIO, and not SPI hardware SS control.  However you would not be able to use the DAC1 anymore for your product if you needed it.

**Comment by monkbroc** [comment]
Is this resolved @technobly?
